test/Interop/Cxx/foreign-reference/witness-table.swift

```diff
@@ -1,6 +1,7 @@
 // RUN: %target-run-simple-swift(-I %S/Inputs/ -Xfrontend -enable-experimental-cxx-interop -Xfrontend -validate-tbd-against-ir=none -Xfrontend -disable-llvm-verify -g)
 //
 // REQUIRES: executable_test
+// REQUIRES: rdar://95738946
```

Comment:
Should be 
`// REQUIRES: rdar95738946`
due to some lit nonsense that I don't understand.
It'll be easier for you to fix this PR than for me to post a new one. There are more urgent problems at the moment.